AF v2 - sample-search artifact argument addition (#6417)
demisto/content: Packs/AutoFocus/Integrations/AutofocusV2/CHANGELOG.md

## [Unreleased]

> Added to ***autofocus-search-samples*** command a new argument, artifact which is set true by default
> Added to ***autofocus-samples-search-results*** artifacts results to context and divided human readable to a new entry for each sample.

Reviewer: ```suggestion
Added to ***autofocus-search-samples*** command the *artifact* argument which is set true by default and retrieves the artifacts of the sample.
```
